person.rb

```diff
@@ -0,0 +1,21 @@
+class Person
+  attr_accessor :name, :age
+  attr_reader :id
+
+  def initialize(age, name = 'Unknown', parent_permission: true)
+    @id = Random.ran(1..1000)
```

Comment:
- [ ] Here you misspelled the method, the method is `Random.rand` instead of `Random.ran`, kindly change this 😄